### DISCUSSION TITLE
Hdoujin info.txt bug ?

### DISCUSSION THREAD

**Discussion by happy1229** [Difegue/LANraragi#507] [Q&A] [answered]
Hdoujin v.0.5 by Pao in the plugin
I can display artist and other information normally with info.json
But if you use info.txt, you can only display tags   
Is this a bug?

**Comment by Difegue** [comment, marked as answer]
Hi,

For both json and txt variants, the plugin only gets what's in the tags field!  
I'm guessing artist and the like are in different fields for the .txt files, I wasn't the one who wrote this plugin initially.
